git-issue.sh

```diff
@@ -36,7 +36,7 @@ IFS=:
 # Set library path
 # shellcheck disable=SC2086
 # Rationale: Word splitting not an issue
-LIB_PATH="$(dirname $0)/../lib:$LD_LIBRARY_PATH:/usr/local/lib:/usr/lib"
+LIB_PATH="$(dirname $0)/../libexec:$LD_LIBRARY_PATH:/usr/local/libexec:/usr/libexec"
```

Comment:
Won't we hard-code `$(libexecdir)/git-issue` and loose the silliness of trying to guess the location?

Reply:
oh yes, of course ...